packages/react-components/react-table/src/hooks/useTableColumnSizing.tsx

```diff
@@ -49,6 +50,23 @@ function useTableColumnSizingState<TItem>(
   const columnResizeState = useTableColumnResizeState(columns, width + (params?.containerWidthOffset || 0), params);
   // Creates the mouse handler and attaches the state to it
   const mouseHandler = useTableColumnResizeMouseHandler(columnResizeState);
+  // Creates the keyboard handler for resizing columns
+  const keyboardResizing = useInteractiveKeyboardResizing(columnResizeState);
+
+  const onMenuItemClick = React.useCallback(
+    (columnId: TableColumnId) => (e: React.MouseEvent) => {
+      e.preventDefault();
+      e.stopPropagation();
+      keyboardResizing.toggleInteractiveMode(columnId);
+    },
+    [keyboardResizing],
+  );
+
+  tableState.accessibilityMenuItems?.push({
+    key: 'column-sizing',
+    label: 'Keyboard Column Resizing',
```

Comment:
How would these labels be internationalized?

Reply:
instead of the user rendering the `item.label`, they can decide what to render based on `item.key`

Reply:
in that case let's not add a default label, we should not be responsible for content. I don't think it's unreasonable to make apps label their menuitems correctly

Reply:
I'm just passing the onClick handler and letting the user label the menu items :)